modules/bcftools/query/main.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process BCFTOOLS_QUERY {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::bcftools=1.13" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/bcftools:1.13--h3a49de5_0"
+    } else {
+        container "quay.io/biocontainers/bcftools:1.13--h3a49de5_0"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
```

Comment:
According to the docs, `query` also accepts the regions/samples/targets file as `view`, so please add as there (also with input specifications)

There seems to be a `-vcf-list` option as well for input... but I guess that's an alternative to supply the VCF file itself... not sure how to deal with that. Maybe worth to ask on #modules.

Reply:
> There seems to be a -vcf-list option as well for input... but I guess that's an alternative to supply the VCF file itself 

Yeah, this is a bit perplexing to me too. As far as I understand, I think it is somewhat similar to the `--samples-file` since it basically lists the VCF file names. I haven't seen this option being used so much 🤔

Reply:
> According to the docs, query also accepts the regions/samples/targets file as view, so please add as there (also with input specifications)



Apart from the `--vcf-list`,  I have accomodated the changes which were made as part of the conversation on https://github.com/nf-core/modules/pull/669

Reply:
OK lets leave it as is for the moment. If someone does use it then they can update the module 👍